### ISSUE TITLE
Rimpy don't  work on arch

### ISSUE THREAD

**Issue by cylian914** [rimpy-custom/RimPy#92]
it don't work on arch, maybe make it open source so we can fix it ? 

![image](https://user-images.githubusercontent.com/73445993/192146871-67c0e41e-bec3-4a45-ae0e-af9964abc125.png)
![image](https://user-images.githubusercontent.com/73445993/192146897-7e271682-267a-49e7-8f90-65f647ed1d74.png)

**Comment by aleksiye** [comment]
Solution (workaround actually):
You have to downgrade your nss (network security services) to version 3.81-1 which is done with a following command
sudo pacman -U https://archive.archlinux.org/packages/n/nss/nss-3.81-1-x86_64.pkg.tar.zst

**Comment by Settyness** [comment]
> Solution (workaround actually): You have to downgrade your nss (network security services) to version 3.81-1 which is done with a following command sudo pacman -U https://archive.archlinux.org/packages/n/nss/nss-3.81-1-x86_64.pkg.tar.zst

I've been trying to get RimPy to work recently on Arch Linux and came across the same error message. I downgraded `nss` and `lib32-nss` and now nothing happens. No terminal output, no logs. I came here to see if anyone had the same issue.

**Comment by hsmalley** [comment]
On Fedora I have to downgrade from 3.8X to 3.7X (nss-3.77.0-1.fc36.x86_64) to get make it work correctly, So rather than downgrade whenever I want to use it, I just downloaded the rpm's for nss & nss-util and extracted the library files into the app's dir. Once I did that everything worked fine

**Comment by jaype87** [comment]
Deleting libnssutil3.so from the rimpy folder works for me. Seems to be a version conflict between the locally installed version of libnss and the packaged libnss-util.

**Comment by heatherathebyne** [comment]
> Deleting libnssutil3.so from the rimpy folder works for me. Seems to be a version conflict between the locally installed version of libnss and the packaged libnss-util.

This also works for OpenSUSE Tumbleweed.

**Comment by tanandry** [comment]
> Deleting libnssutil3.so from the rimpy folder works for me. Seems to be a version conflict between the locally installed version of libnss and the packaged libnss-util.

Also work in Manjaro.

**Comment by Pangaea666** [comment]
> Deleting libnssutil3.so from the rimpy folder works for me. Seems to be a version conflict between the locally installed version of libnss and the packaged libnss-util.

Thank you so much for this suggestion. It also worked on my system, which is Linux Mint.

**Comment by rimpy-custom** [comment]
Try latest prerelease

**Comment by Adda0** [comment]
The above-mentioned problem was resolved for me (Arco Linux, Arch-based distribution) with the last [prerelease 1.2.6.28](https://github.com/rimpy-custom/RimPy/releases/tag/1.2.6.28). RimPy works as of now without the need to manually delete ` libnssutil3.so` library.

Nevertheless, I still get the following errors immediately after starting the app, but at least the app runs out-of-the-box now.
```
$ ./RimPy.sh
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Sandboxing disabled by user.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
11.382188034998762
js: Uncaught ReferenceError: qt is not defined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'parentElement' of null

$ ./RimPy
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Sandboxing disabled by user.
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
13.035899450000215
js: Uncaught ReferenceError: qt is not defined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'parentElement' of null
```

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
